scanner/src/main/kotlin/Scanner.kt

```diff
@@ -158,4 +166,22 @@ abstract class Scanner(val scannerName: String, protected val config: ScannerCon
             data += ortResult.data
         }
     }
+
+    /**
+     * Filter the scan results in the [resultContainer] for only license findings that are in the same subdirectory as
+     * the [project]s definition file.
+     */
+    private fun filterProjectScanResults(project: Project, resultContainer: ScanResultContainer): ScanResultContainer {
+        val pathInRepository = project.definitionFilePath.substringBeforeLast("/")
```

Comment:
Is the the path in the repository or is it the path relative to the scan directory
(which is a different thing in case of Git Rep)?

Reply:
The definition file paths are always relative to the repository.